js/gated-content/src/components/MediaPlayer.vue

```diff
@@ -6,12 +6,11 @@
       :videoId="videoId"
       :options="{responsive: 'true', url: media.field_media_video_embed_field}"
       :player-vars="handleAttributes()"
-      :autoplay="autoplay"
-      @loaded="handleLoaded()"
-      @ready="handleReady()"
+      @loaded="setTimeCode()"
```

Comment:
You shouldn't just change existing callbacks to new ones. Please, put your code into existing methods: handleLoaded and handleReady. Also, keep in mind that we should autoplay videos only if the appropriate setting is enabled.

Reply:
We have updated the code. Can you please review once.